gravatar-quickeditor/src/main/java/com/gravatar/quickeditor/ui/alttext/AltTextPage.kt

```diff
@@ -175,7 +185,26 @@ internal fun AltTextPage(
                         textStyle = LocalTextStyle.current.copy(color = MaterialTheme.colorScheme.onSurface),
                         modifier = Modifier
                             .fillMaxWidth()
-                            .padding(start = 16.dp, top = 8.dp, bottom = 8.dp),
+                            .fillMaxHeight(),
+                        decorationBox = { innerTextField ->
```

Comment:
This makes it so that the whole `decorationBox` acts as a touch are - meaning we can tap everywhere to focus the text field.

We could also use this for the placeholder I think.